addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2958,6 +2958,22 @@ options.registry.HideFooter = VisibilityPageOptionUpdate.extend({
     shownValue: 'shown',
 });
 
+options.registry.ContainerWidthFooter = options.registry.ContainerWidth.extend({
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+    /**
+     * Allow to customize the view only if the footer copyright section
+     * is the target (due to the data-apply-to)
+     * @override
+     */
+    customizeWebsiteViews: async function (previewMode, widgetValue, params) {
```

Comment:
You should prefer the form `async customizeWebsiteViews(previewMode, widgetValue, params) {` instead of the use of the `function` keyword. That will be required when it will be a real JS class with the new OWL system.